ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -151,3 +151,52 @@ def test_smooth_l1_loss(
         beta=beta,
         reduction=reduction,
     )
+
+
+# huber_loss
+@handle_test(
+    fn_tree="functional.ivy.experimental.huber_loss",
+    dtype_and_true=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Can you try changing these to "valid" instead so we expand the scope of the tests? I've given it a try and it doesn't seem to add too many difficulties. it didn't cause any of the passing tests to fail